pages/common/kubetail.md

```diff
@@ -0,0 +1,19 @@
+# kubetail
+
+> Utility to tail multiple Kubernetes pod logs at the same time.
+
+- Tail the logs of multiple pods (which name starts with "my_app") in one go:
+
+`kubetail {{my_app}}`
+
+- Tail only a specific container from multiple pods:
+
+`kubetail my_app -c {{my_container}}`
```

Comment:
Perhaps `my_app` should be wrapped in `{{` and `}}`?

Reply:
I agree